code/controllers/subsystem/job.dm

```diff
@@ -594,10 +594,12 @@ SUBSYSTEM_DEF(job)
 /datum/controller/subsystem/job/proc/LoadJobs()
 	var/jobstext = file2text("[global.config.directory]/jobs.txt")
 	for(var/datum/job/job as anything in joinable_occupations)
-		var/regex/jobs = new("[job.title]=(-1|\\d+),(-1|\\d+)")
+		var/regex/jobs = new("[job.title]=(-1|\\d+),(-1|\\d+),(-1|\\d+),(-1|\\d+)")
```

Comment:
what happens if they have old jobs.txt?

Reply:
A "cannot read from list" runtime will happen and the values, including jobslots, will be their code-dictated default.

Reply:
Make the new config be called `jobconfig.txt` and keep the old loader but only call it if jobconfig.txt is not found.

You can not change the format of a config file or otherwise create a need to do a thing before updating the game code because most servers do auto updates and also we have synced configs but also multiple servers that can't all be running the same code at the same time given rounds end at different points in time.